diff --git a/nearcore/src/dyn_config.rs b/nearcore/src/dyn_config.rs
index 4a9814231..f44cfda58 100644
--- a/nearcore/src/dyn_config.rs
+++ b/nearcore/src/dyn_config.rs
@@ -45,6 +45,7 @@ pub fn read_updateable_configs(
 pub fn get_updateable_client_config(config: Config) -> UpdateableClientConfig {
     // All fields that can be updated while the node is running should be explicitly set here.
     // Keep this list in-sync with `core/dyn-configs/README.md`.
+    crate::metrics::EXPECTED_SHUTDOWN_BLOCK.set(config.expected_shutdown.unwrap_or_default() as i64);
     UpdateableClientConfig { expected_shutdown: config.expected_shutdown }
 }
 
diff --git a/nearcore/src/metrics.rs b/nearcore/src/metrics.rs
index bfadbe9c6..e0db5ad14 100644
--- a/nearcore/src/metrics.rs
+++ b/nearcore/src/metrics.rs
@@ -99,3 +99,9 @@ pub(crate) static STATE_SYNC_OBTAIN_PART_DELAY: Lazy<near_o11y::metrics::Histogr
         )
         .unwrap()
     });
+
+pub(crate) static EXPECTED_SHUTDOWN_BLOCK: Lazy<IntGauge> = Lazy::new(|| {
+    try_create_int_gauge("near_block_expected_shutdown", "The block height expected shutdown")
+    .unwrap()
+});
+
